packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -3464,6 +3613,46 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin {
     return push<T>(_routeNamed<T>(routeName, arguments: arguments));
   }
 
```

Comment:
the restorable* methods should be mentioned in their corresponding non-restorable version